.github/stale.yml

```diff
@@ -1,4 +1,4 @@
-daysUntilStale: 540 # 1.5Y - 7d
+daysUntilStale: 358 # 1.5Y - 7d
```

Comment:
nit: change comment to `1Y - 7d`

Reply:
oops.  can't even manage a 1 line change 😅